### PR TITLE
Mention that annotations in GDScript support constant expressions

### PR DESCRIPTION
**Pull request by dalexeev**
According to godotengine/godot#71634.

### CODE REVIEW

File: tutorials/scripting/gdscript/gdscript_basics.rst [hunk 1]
```diff
@@ -338,17 +338,24 @@ For instance, you can use it to export a value to the editor::
 For more information about exporting properties, read the :ref:`GDScript exports <doc_gdscript_exports>`
 article.
 
+Any constant expression can be passed as an annotation argument::
```
Comment (dalexeev) [inline tutorials/scripting/gdscript/gdscript_basics.rst:341]:
```suggestion
Any constant expression that is compatible with the required argument type can be passed as an annotation argument value::
```

**Comment by dalexeev** [comment]
Are there any suggestions for this PR?

File: tutorials/scripting/gdscript/gdscript_basics.rst (hunk 1 above)
Comment (Piralein) [reply tutorials/scripting/gdscript/gdscript_basics.rst:341]: I would remove `that is` . But I'm fine with each of the suggested sentences.

Comment (dalexeev) [reply tutorials/scripting/gdscript/gdscript_basics.rst]: Done.

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Nope, in this goes :) Thank you! 🎉

**Pull request merged**